conmon/conmon.c

```diff
@@ -156,6 +156,13 @@ static int log_fd = -1;
 		exit(EXIT_FAILURE); \
 	} while (0)
 
+#define pwarn(s) \
+	do { \
+		fprintf(stderr, "[conmon:e]: %s %s\n", s, strerror(errno)); \
```

Comment:
Might want to change this to `[conmon:w]` since it is a warning.

Reply:
Also do we want t check if the log level is set to warn and above, or do we just always want to log this?

Reply:
We always want to log this.